packages/react-dom/src/shared/DOMNamespaces.js

```diff
@@ -31,6 +35,9 @@ export function getChildNamespace(
   parentNamespace: string | null,
   type: string,
 ): string {
+  if (enableEventAPI && type === REACT_EVENT_COMPONENT_TYPE) {
```

Comment:
You can revert this whole block because it can't happen anymore. The `type` is a string and `REACT_EVENT_COMPONENT_TYPE` is a symbol.